docs/AxilrodTellerPotential.md

```diff
@@ -0,0 +1,35 @@
+# Axilrod-Teller(-Muto) Potential
+
+[Reference Paper](https://pubs.aip.org/aip/jcp/article/11/6/299/182086/Interaction-of-the-van-der-Waals-Type-Between)
+
+![3-body-sketch](./graphics/3_body_sketch.png)
+
+The Axilrod-Teller potential is a model for the interactions of three molecules which appear when the van der Waals forces are approximated to the third order.
+
+$$ U_{AT} = \nu \frac{3 \cos\gamma_1 \cos\gamma_2 \cos\gamma_3 + 1}{r_{12}^3 r_{23}^3 r_{31}^3} $$
+, where $r_{ij}$ is the distance between particles $i$ and $j$ and $\gamma_i$ is the angle between the sides $r_{ij}$ and $r_{ik}$. $\nu$ is a material dependent parameter of the order $V\alpha^3$, where $V$ is the ionization energy and $\alpha$ the polarizability.
+
+The cosines can also be expressed as:
+$$\cos\gamma_1 = \frac{\vec{r}_{12} \cdot \vec{r}_{13}}{|\vec{r}_{12}||\vec{r}_{13}|}$$
+, where $\vec{r}_{ij}$ is the vector from particle $i$ to particle $j$ ($i \longrightarrow j$ ).
```

Comment:
```suggestion
The cosines can also be expressed as:

$$
\cos\gamma_1 = \frac{ \vec{r}_ {13} \cdot \vec{r}_ {13}}{|\vec{r}_ {12}||\vec{r}_ {13}|}
$$

, where $\vec{r}_{ij}$ is the vector from particle $i$ to particle $j$ ($i \longrightarrow j$ ).
```
Funky things are happening... This one only works if you either escape the underscores or put a space behind them